dataactbroker/README.md

```diff
@@ -773,6 +773,153 @@ Example output if there are no files available:
 }
 ```
 
+# File Generation Routes
+
+## Generate Files
+**Route:** `/generate_file`
+
+**Method:** `POST`
+
+This route sends a request to the backend to utilize the relevant external APIs and generate the relevant file for the metadata that is submitted.
+
+**Deprecation Notice:** This route replaces `/generate_d1_file` and `/generate_d2_file`.
```

Comment:
Thanks for being explicit about what's deprecated. This raises a good question that I don't think we need to answer right now, but I'll create a post-launch story for it...how will we handle API versioning?